pkg/apis/migration/v1alpha1/model.go

```diff
@@ -23,6 +26,11 @@ const (
 	MigplanMigrationFailed  = "migplan.migration.openshift.io/failed"
 )
 
+// Environment variable for disable_image_copy flag
+const (
```

Comment:
This should probably be loaded in through the `settings` pkg.

Reply:
Moved the function to settings pkg.